license-information-windows.md

```diff
@@ -2,30 +2,25 @@
 
 The Windows distribution of .NET contains files that are provided under multiple licenses.
 This information is provided to help you understand the license terms that
-apply to your use. By using any or all of these files you agree to their associated license terms.
-
-See [license information](./license-information.md) for information about other operating systems.
+apply to your use. By using .NET on Windows, you agree to the following license terms.
 
 This document is provided for informative purposes only, and is not itself a license.
 
-The following binaries are licensed with the
-[Windows SDK License](https://learn.microsoft.com/legal/windows-sdk/license):
-
-* api-ms-\*.\* (used by .NET runtime, .NET 6 and earlier)
-* ucrtbase.dll (used by .NET runtime, .NET 6 and earlier)
-* D3DCompiler_47_cor3.dll (used by WPF)
-
 The following binaries are licensed with the
 [.NET Library License](https://dotnet.microsoft.com/dotnet_library_license.htm)
 
+* coreclr.dll and .NET runtimes included in binaries published as single-file (due to extra telemetry included by .NET runtime in Watson crash reports)
```

Comment:
We also need to include `vcruntime140_cor3.dll (used by WPF)` that started all this discussion in the .NET Library License list. For extra clarity, we may add a note that it is same binary as `vcruntime140.dll` included in Visual Studio redist relicensed under .NET Library License with Microsoft permission.